napari_plugin_manager/_tests/test_qt_plugin_dialog.py

```diff
@@ -253,30 +253,25 @@ def test_filter_installed_plugins(plugin_dialog):
     assert plugin_dialog.installed_list._count_visible() == 0
 
 
-def test_visible_widgets(plugin_dialog):
+def test_visible_widgets(request, plugin_dialog):
     """
-    Test that the direct entry button and textbox are visible for
-    normal napari installs.
+    Test that the direct entry button and textbox are visible
     """
-
+    if "no-constructor" not in request.node.name:
+        # the plugin_dialog fixture has this id
+        # skip for 'constructor' variant
+        pytest.skip()
     assert plugin_dialog.direct_entry_edit.isVisible()
     assert plugin_dialog.direct_entry_btn.isVisible()
 
 
-def test_constructor_visible_widgets(plugin_dialog_constructor):
-    """
-    Test that the direct entry button and textbox are hidden for
-    constructor based napari installs.
-    """
-    assert not plugin_dialog_constructor.direct_entry_edit.isVisible()
-    assert not plugin_dialog_constructor.direct_entry_btn.isVisible()
-
-
-def test_version_dropdown(plugin_dialog):
+def test_version_dropdown(qtbot, plugin_dialog):
     """
     Test that when the source drop down is changed, it displays the other versions properly.
     """
     widget = plugin_dialog.available_list.item(1).widget
+    qtbot.wait(300)
+
```

Comment:
Why wait, not wait until? I do not see here triggering any actions.

Reply:
Oops, might be an oversight. Let's remove it, thanks!

```suggestion
```